Source/Provers/SMTLib/ProverUtil.cs

```diff
@@ -82,6 +84,7 @@ public virtual string Help
 MEMORY_LIMIT=<int>        Memory limit of the prover in megabytes.
 VERBOSITY=<int>           The higher, the more verbose.
 TIME_LIMIT=<uint>          Time limit per verification condition in milliseconds.
+BATCH_MODE=<bool>         If true, run in batch mode rather than interactive mode.
```

Comment:
Consider adding a "See here for details" link of some kind, since "batch mode" is not a standard solver concept AFAICT.

Reply:
I reworded it in a way that's hopefully more self-explanatory.

This is perhaps an argument for Boogie to have some documentation, though. :) There's currently none in the repository.